src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -89,6 +95,14 @@ public TenantCollection getAll(UUID consortiumId) {
     return result;
   }
 
+  @Override
+  public TenantDetails getTenantDetailsById(UUID consortiumId, String tenantId) {
+    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
+    var tenantDetailsEntity = tenantDetailsRepository.findById(tenantId).orElseThrow(() ->
+      new ResourceNotFoundException("id", tenantId));
```

Comment:
It would be great to specify what storts of id not found. Because, error message starts with Object not found with id[] 
for example to change tenantDetailsId or tenantId.

Reply:
did this consistent with resource not found messages for the tenant in this service. I think we can address it in another pr

Reply:
updated